client/ionic/src/content/userContext.ts

```diff
@@ -0,0 +1,19 @@
+// TODO: Product to spec what facets of a user control the content they see.
+export default interface UserContext {
+  // lowercase ISO 3166-2 region code or the special designator 'ww' for worldwide
+  country: string;
+  // BCP 47 language tag (ex. en, fr-CA, etc.).  Note the country here (if any) is
+  // a linguistic dialect, separate from the country above.
+  // TODO: What about back off from language with dialect, to without dialect (fr-CA -> fr)?
```

Comment:
it may be overkill, but rather than blanket backing off from dialect, you could include it in the waterfall of fallbacks

Reply:
Sorry we may be using the terms backoff and fallback differently.  Without the TODO, the current behavior is if you have Canadian French as language it would try Canadian French then English.  After the TODO it would be Canadian French, French, English.

(This is independent of the country of location ofc).

Reply:
thanks for clarifying, we are on the same page 👍